ArduCopter/Parameters.cpp

```diff
@@ -1025,6 +1025,27 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("PILOT_SPEED_DN", 24, ParametersG2, pilot_speed_dn, 0),
 
+    // @Param: FS_THR_OPTIONS
+    // @DisplayName: Radio failsafe options bitmask
+    // @Description: Bitmask of additional options for radio failsafe
+    // @Values: 0:Continue when in Auto mode, 1:Continue when in Guided mode, 2:Continue landing in Land mode
```

Comment:
do not use `@Values:` use `@Bitmask:` instead. The values themselves are correct already, no need to change those